ui/server/src/main/scala/pl/touk/nussknacker/ui/process/migrate/ProcessModelMigrator.scala

```diff
@@ -15,7 +17,7 @@ case class MigrationResult(process: CanonicalProcess, migrationsApplied: List[Pr
   def toUpdateAction(processId: ProcessId): UpdateProcessAction = UpdateProcessAction(
     id = processId,
     canonicalProcess = process,
-    comment = s"Migrations applied: ${migrationsApplied.map(_.description).mkString(", ")}",
+    comment = SystemComment(s"Migrations applied: ${migrationsApplied.map(_.description).mkString(", ")}"),
```

Comment:
We can have `MigrationComment(migrationsApplied: List[ProcessMigration]) <: SystemComment` with method `def message = "..."`. Thanks to that will be easier to find all system comments and presentation logic will be split from business logic and will be easier to test it

Reply:
sorry I missed this comment before, just noticed now when trying to merge, can you take a look?